Content.Shared/Teleportation/Systems/SwapTeleporterSystem.cs

```diff
@@ -167,11 +181,48 @@ public void DoTeleport(Entity<SwapTeleporterComponent, TransformComponent> ent)
             return;
         }
 
+
+        // Prevents teleporting to the polymorph zone or the cryosleep zone.
+        // Don't unlink because it might be temporary (polymorph). Let them figure it out and unlink it themselves.
+        if (_map.IsPaused(Transform(teleEnt).MapID) || _map.IsPaused(Transform(otherTeleEnt).MapID))
+        {
+            _popup.PopupEntity(Loc.GetString("swap-teleporter-popup-teleport-fail",
+                ("entity", Identity.Entity(linkedEnt, EntityManager))),
+                teleEnt,
+                teleEnt,
+                PopupType.MediumCaution);
+            return;
```

Comment:
these conditions really all ought to be combined into a single check called "CanTeleport" or smth.

Reply:
Done